mods/content/polaris/datum/factions.dm

```diff
@@ -1,9 +1,211 @@
 /decl/background_detail/faction/nanotrasen
-	name = "NanoTrasen"
+	name = "Corporate - NanoTrasen"
 	uid = "faction_nanotrasen"
 	description = "NanoTrasen is one of the leading research and development companies in Solar space. \
 	Originally focused on consumer products, their early move into Phoron research and extraction \
 	made them the foremost experts on the substance and its uses. \
 	Nanotrasen also offers one of the most comprehensive medical plans in SolGov space, \
 	up to and including cloning, prosthetics and therapy. \
 	Some have accused the company of using its employees as 'guinea pigs' for new technologies."
+	economic_power = 1.2
+	subversive_potential = 0.8
+
+
+/decl/background_detail/faction/contractor
```

Comment:
You know what, this is probably more sensible than listing every corporation individually unless we do anything else with the datums later. Good thinking.

Reply:
i can't take credit for it, this is how it was on the etherpad